freeipa.spec.in

```diff
@@ -993,6 +993,11 @@ if [  $? -eq 0 ]; then
     if [  $? -eq 0 ]; then
         /bin/systemctl restart ipa.service >/dev/null
     fi
+
+    /bin/systemctl is-enabled ipa-ccache-sweep.timer >/dev/null 2>&1
+    if [  $? -eq 1 ]; then
+        /bin/systemctl enable ipa-ccache-sweep.service >/dev/null
```

Comment:
You are enabling the service but not the timer. How it supposed to work? When timer is triggered, it will start the service but if timer is not enabled, it will not trigger.

Reply:
Gah, good catch. Yes, in the case of a systemd timer only the timer needs to be enabled, not the service. Fixed.